### PR TITLE
Many small fixes

### PR DESCRIPTION
**Pull request by sidpremkumar**
* Fix goroutine issue and ensure ws are closed
* Fix naming and file formating to be consistent with lucies design
* Keep pattern consistent between server/daemon for files

## Description of the change

> Description here (why is it needed, what does it do)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Refactor (code change with no functionality change)
- [ ] Enhancement (minor change below the level of a feature)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Remove feature (removes a feature we don't support anymore)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related JIRA tickets

Relates to JIRA: CWC-942 CWC-954 CWC-947 CWC-937

## Relevant release note information

Release Notes:

## Potential Security Impacts

Does this PR have any security impact?
Does it fix a security issue?
Does it add attack surface?
Why do we think this change is safe?

### Development

- [ ] Does the code changed/added as part of this pull request have test coverage?
- [ ] Do all tests related to the changed code pass in development?
- [ ] Have you ensured that at least one other person has tested your code?
- [ ] Have you tested the code?
- [ ] Have you attached any pictures (if relevant)?

### CODE REVIEW

File: bctl-go-daemon/CommonWebsocketClient/main.go [hunk 1]
```diff
@@ -42,33 +48,44 @@ func NewCommonWebsocketClient(serviceUrl string, hubEndpoint string, params map[
...
-					message = message[0 : len(message)-1]
-				}
-
-				// Also check to see if we have multiple messages
-				seporatedMessages := bytes.Split(message, []byte{messageTerminator})
-
-				for _, formattedMessage := range seporatedMessages {
-					// And alert on our channel
-					ret.WebsocketMessageChan <- formattedMessage
+			select {
```
Comment (sidpremkumar) [inline bctl-go-daemon/CommonWebsocketClient/main.go:59]: This was taken from @ymarcus93s code: https://github.com/bastionzero/bzero-ssm-agent/blob/bzero-dev/agent/session/plugins/fileuploaddownload/fileuploaddownload.go#L180

It allows us to have a cancel context and it seems to be best practice.

File: bctl-go-daemon/Daemon/daemonWebsocket/plugins/handleExec/handleExec.go [hunk 2]
```diff
@@ -157,99 +147,122 @@ func HandleExec(w http.ResponseWriter, r *http.Request, wsClient *DaemonWebsocke
...
 
 	// Set up a go function for stdin
 	go func() {
-		buf := make([]byte, 16)
 		for {
-			n, err := proxy.stdinStream.Read(buf)
-			// Handle error
-			if err == io.EOF {
-				// TODO: This means to close the stream
+			buf := make([]byte, 16)
```
Comment (tflpd) [inline bctl-go-daemon/Daemon/daemonWebsocket/plugins/handleExec/handleExec.go:213]: Why did this get moved inside the `for` loop?

File: bctl-wrapper/bctl/main.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import subprocess
+
+import uuid
+
+def main():
+    # Extract the args
+    argsForKube = f'{" ".join(sys.argv[1:])}'
+
+    # Append the args to the token in the kubeconfig using the zli
+    token = subprocess.check_output('/Users/sidpremkumar/Documents/CommonwealthCrypto/zli/bin/zli-macos --configName dev get-kube-token -s', shell=True).decode('utf-8').strip()
```
Comment (tflpd) [inline bctl-wrapper/bctl/main.py:11]: Can we make this path automatically configured?

File: src/handlers/get-kube-token.handler.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+
+import { config } from 'yargs';
+import { ConfigService } from '../config.service/config.service';
+
+
+const ONE_HOUR = 60 * 60 * 1000; /* ms */
```
Comment (tflpd) [inline src/handlers/get-kube-token.handler.ts:6]: Is this used somewhere?

**Review by tflpd — APPROVED** [review]

File: bctl-go-daemon/Daemon/daemonWebsocket/plugins/handleExec/handleExec.go (hunk 2 above)
Comment (sidpremkumar) [reply bctl-go-daemon/Daemon/daemonWebsocket/plugins/handleExec/handleExec.go]: https://github.com/bastionzero/zli/pull/126/commits/cf7c57b8425585f29eee8523530d67c5aadd1384

File: bctl-wrapper/bctl/main.py (hunk 3 above)
Comment (sidpremkumar) [reply bctl-wrapper/bctl/main.py:11]: this will point to zli

File: src/handlers/get-kube-token.handler.ts (hunk 4 above)
Comment (sidpremkumar) [reply src/handlers/get-kube-token.handler.ts]: https://github.com/bastionzero/zli/pull/126/commits/cf7c57b8425585f29eee8523530d67c5aadd1384

**Pull request merged**